runtime_config.go

```diff
@@ -1,8 +1,10 @@
 package specs
 
 type RuntimeSpec struct {
-	// Mounts profile configuration for adding mounts to the container's filesystem.
-	Mounts []Mount `json:"mounts"`
+	// Mounts is a mapping of names to mount configuration.
+	// Which mounts will be mounted and where should be chosen with MountPoints
+	// in Spec.
```

Comment:
It's unfortunate that `*Spec` didn't change to `*Config` as part of 7232e4b (specs: introduce the concept of a runtime.json, 2015-07-30, #88), but that's probably orthogonal to this PR ;).